docs/configuration/OVERVIEW.md

```diff
@@ -36,103 +36,147 @@ A component will have to handle different types of configurations.
 
 ### Default configuration
 
-- Each __component type__ will have a default configuration which will be defined in _.config.ts_ file associated to the
-  component
+- Each __configurable component__ will have a default configuration which will be defined in the `.config.ts` file associated to the component
 
-> __WARNING__ the field name 'id' should not be used in the configuration, as we created a unique one for the entity configuration store
+> [!WARNING]
+> The field name `id` should not be used in the configuration, as we created a unique one for the entity configuration store.
 
-### Configuration coming from config store
+### Configuration coming from the configuration store
 
-- Each __component type__ can have a customized config (compliant with the store model). This custom config can be
+- Each __configurable component__ can have a customized configuration (compliant with the store model). This custom configuration can be
   asynchronously loaded from a backend server, or it can be injected in the body tag of the app. In both cases, this
   content will be used to update the store.
-- If present, it will override the default config. (Don't worry about the implementation now, we'll see it later)
+- If present, it will override the default configuration. Don't worry about the implementation now, we'll see it later.
 
-### Global config
+### Global configuration
 
-- The common configuration is the one used in multiple components (it can be a date format, price display, type of input
-  form fields from an Angular material input element ...)
-- A common configuration is defined in every library. The application common configuration (__global config__) will be
-  the result of the __merge of all common__ configurations.
+- The __common configuration__ is the one used in multiple components. It can be a date format, a price display, an input
+  form field type from an Angular material input element, etc.
+- A common configuration is defined in every library. The application's common configuration (__global configuration__) will be
+  the result of the __merge of all the common__ configurations.
 - The common configuration is __not overridden__ at component type (there will be no properties with the same name in
-  common config and components).
-- At runtime after the configuration of a component is computed it is extended by global config, so components can use
-  fields from global
+  common configuration and components).
+- At runtime, after the configuration of a component is computed, it is extended by the global configuration, so components can use
```

Comment:
comment from @cpaulve-1A:
Missing comma line 60 and I am not sure I understand what it means. An example could be great there

https://github.com/AmadeusITGroup/otter/pull/1583/files#r1555920906